contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -41,8 +41,13 @@ func deposit_handled(l2_token : felt, account : felt, amount : Uint256):
 end
 
 @event
-func bridged_rewards(l2_token: felt, acocunt: felt, amount: Uint256):
+func mint_rewards_initiated(l2_reward_token : felt, account : felt, amount : Uint256):
```

Comment:
I think the event would make more sense as `rewards_minted` or just `mint_rewards` because there is no 'initiation', when this event is emitted the rewards have been minted.

Reply:
I agree!